### ISSUE TITLE
Google-research repo (with e.g Julia)

### ISSUE THREAD

**Issue by PallHaraldsson** [d2s/companies#124]
FYI Google has another repo you don't have. I didn't add Julia because it's not on their main repo. There might be more languages than Julia there missing from the main one:

https://github.com/google-research/FirstOrderLp.jl

**Comment by d2s** [comment]
Yeah, I haven't updated the list in a long time, so it's most likely outdated in many parts.

Have been thinking that some kind of automated way to pull contents of _Languages_ dropdown from listed GitHub Organizations would enable faster and easier updating of the list.
![2022-04-08 01 14 30 github com d12f8c99a9a1 - Google Research - Languages dropdown](https://user-images.githubusercontent.com/135053/162328443-edb11c2e-d86e-4c41-bcd4-2500ebc7004f.png)

Basically have been thinking of a small script that would search a DOM selector to get the text content from the dropdown element, and reformat it to a comma separated list. I know it is relatively easy to create, but just haven't had energy to build it.

**Comment by d2s** [comment]
As the repo & website is now updated to include Google Research organization, closing this issue.
Created another issue to track ideas about automated list updating ideas:
- https://github.com/d2s/companies/issues/125
